### DISCUSSION TITLE
Need Help with Swap icons with rotate effect

### DISCUSSION THREAD

**Discussion by myhendry** [saadeghi/daisyui#531] [Q&A] [answered]
Hi. I'm trying out the component Swap icons with rotate effect

I copied and paste the html code into my react tsx file and it shows all the individual components including the moon, star and checkbox

What should I do to make it work?

Thanks

**Comment by saadeghi** [comment, marked as answer]
I think you're using an older version of daisyUI. upgrade to latest version. `swap` [was added in version `2.0`](https://daisyui.com/docs/changelog/)

**Reply by myhendry** [reply]
Thanks! It works
